bin/install

```diff
@@ -203,6 +204,13 @@ build() {
   yarn
   yarn build --ignore=@aws-lambda/*
   yarn image
+
+  if [ "${configs[image-repository]}" ]; then
+    for i in $(docker images --filter "reference=mds*latest" | grep mds | tr -s ' ' | cut -d ' ' -f 1,2 | tr ' ' ':'); do
```

Comment:
Can use `docker images --format '{{ .Repository}}:{{ .Tag}}' --filter 'reference=mds-*` inside the `$()`